### PR TITLE
Feature/App-Version

### PR DESCRIPTION
**Pull request by zenangst**

### CODE REVIEW

File: Source/Versions.swift [hunk 1]
```diff
@@ -14,6 +14,19 @@ public enum Semantic {
   case Major, Minor, Patch, Same, Unknown
 }
 
+public class App {
```
Comment (kostiakoval) [inline Source/Versions.swift]: use struct :)

File: VersionsDemo/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
+//
```
Comment (kostiakoval) [inline VersionsDemo/ViewController.swift:1]: remove unused demo file :)

File: Source/Versions.swift [hunk 3]
```diff
@@ -14,6 +14,19 @@ public enum Semantic {
...
+
+  static var version: String = {
+    var version: String = ""
```
Comment (kostiakoval) [inline Source/Versions.swift:20]: should i return "" string or maybe `String?`

**Comment by kostiakoval** [comment]
Yes this is what I wanted :+1:

File: Source/Versions.swift (hunk 3 above)
Comment (zenangst) [inline Source/Versions.swift:20]: Maybe `String?` Is better :)

File: Source/Versions.swift (hunk 1 above)
Comment (zenangst) [inline Source/Versions.swift]: Went with class because of Objective-C compatibility but I guess it's time to drop that.

Comment (zenangst) [inline Source/Versions.swift]: @kostiakoval Fixed

File: VersionsDemo/ViewController.swift (hunk 2 above)
Comment (zenangst) [inline VersionsDemo/ViewController.swift:1]: I updated the demo to actually use the `App.version`, what do you think?

File: Source/Versions.swift (hunk 3 above)
Comment (kostiakoval) [inline Source/Versions.swift:20]: I think string is ok, because you have to specify version when releasing an app.

**Pull request merged**